packages/fortinet_fortiproxy/data_stream/log/fields/fields.yml

```
@@ -1201,6 +1201,10 @@
       type: keyword
       description: >-
         SSH login Name
+    - name: logver
+      type: integer
```

```suggestion
      type: long
```

Long would probably be a better choice, unless the docs specifically call out that it's a 32-bit integer.